vignettes/Pagination.Rmd

```diff
@@ -0,0 +1,229 @@
+---
+title: "Pagination"
+author: "Emily de la Rua"
+date: "`r Sys.Date()`"
+output: 
+  rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Pagination}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{css, echo=FALSE}
+pre {
+  max-height: 800px !important;
+}
+```
+
+## Introduction
+
+This vignette shows how pagination can be applied to `listing_df` objects using the `rlistings` R package.
+
+Specifically, the following topics will be covered:
+
+- Basics of pagination
+- Pagination with exporters
+- Pagination by parameter
+
+To learn more about how listings are constructed using the `rlistings` package, see the [Getting Started vignette](rlistings.html).
+
+---------
+
+## Basics of Pagination
+
+In many cases, listings more rows or more columns than fit on a single page. When this occurs, we may choose to use pagination to improve the readability of the listing and/or to fit in onto a standard-sized page for exporting or printing. When pagination is applied, all listing annotations (titles & footers) are printed on each of the returned page. Note that currently only mono-spaced fonts are supported for pagination.
+
+Vertical pagination occurs when a listing is especially long and has too many _rows_ to fit on one page, while horizontal pagination is applied when a listing is especially wide and has too many _columns_ to print on a single page. When horizontal pagination is applied to a listing, any key columns that have been specified for the listing will be repeated as the leftmost columns on each page. If a value from a key column, for example a subject ID, is present in rows that extend over several pages, that key column value will be printed at the top of each page.
+
+The `paginate_listing` function paginates a listing object and returns a list of listing objects with each list element corresponding to a separate page. This function should be used if you want to paginate a listing to view within your R console or you would like to process the list of paginated output further before exporting.
```

Comment:
btw this does something completely different from what happens in `export_as_txt`, hence the multiple inconsistencies of `export_as_txt`